techniques/html/H28.html

```diff
@@ -18,6 +18,9 @@ <h2>When to Use</h2>
       <h2>Description</h2>
       <p>The objective of this technique is to provide expansions or definitions for
             abbreviations by using the <code class="language-html">abbr</code> element. It is always appropriate to use the <code class="language-html">abbr</code> element for any abbreviation, including acronyms and initialisms.</p>
+            <p>This technique has been altered from a sufficient technique to an advisory technique because it uses the <code>title</code> attribute of <code>abbr</code> to provide the expansion of the abbreviation needed to achieve a sufficient technique when used in combination with <a href="../general/G102">G102</a>. As noted in the <a href="https://html.spec.whatwg.org/multipage/dom.html#the-title-attribute"> specification</a>:
```

Comment:
```suggestion
            <p>This technique has been altered from a sufficient technique to an advisory technique because it uses the <code>title</code> attribute of <code>abbr</code> to provide the expansion of the abbreviation needed to achieve a sufficient technique when used in combination with <a href="../general/G102">G102</a>. As noted in the <a href="https://html.spec.whatwg.org/multipage/dom.html#the-title-attribute"> specification</a>:</p>
```